postgres/README.md

```diff
@@ -130,10 +157,10 @@ PostgreSQL default logging is to `stderr` and logs do not include detailed infor
     ```yaml
       logs:
           - type: file
-            path: /var/log/pg_log/pg.log
+            path: </path/to/logfile>
             source: postgresql
             sourcecategory: database
-            service: myapp
+            service: <myapp>
```

Comment:
I don't think these changes go well with the introduction line - it's not enough anymore to "add this block", users need to edit it. So maybe switch the sentence to "Add and edit this configuration block […]"?